scripts/install_typst.sh

```
@@ -0,0 +1,85 @@
+    esac
+}
+
+TYPST_ARCH=$(get_arch)
+
+wget -qO typst.tar.xz \
+     https://github.com/typst/typst/releases/download/v${TYPST_VERSION}/typst-${TYPST_ARCH}.tar.xz
+
+tar -xf typst.tar.xz
+
+mkdir -p ${INSTALL_PATH}
+mv typst-$(TYPST_ARCH)/typst ${INSTALL_PATH}/typst
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)